onyo/cli/get.py

```diff
@@ -21,8 +21,8 @@
         required=False,
         default=0,
         help=r"""
-            Descend up to **DEPTH** levels into the directories specified. A
-            depth of **0** descends recursively without limit.
+            Descend up to **DEPTH** levels into the directories specified by
+            ``include``. A depth of **0** descends recursively without limit.
```

Comment:
Not sure if we want to mention both, but `--path` is an alias for `--include` and works the same way.

Reply:
IMO that would increase noise and reduce the comprehension. `--path` is indeed that, just an alias.